Makefile

```diff
@@ -104,5 +109,9 @@ gen_patch_iscp:
 gen_patch_values:
 	diff -u pkg/apis/istio/v1alpha2/values/values_types.pb.go.orig pkg/apis/istio/v1alpha2/values/values_types.pb.go > pkg/apis/istio/v1alpha2/values/fix_values_structs.patch || true
 
-vfsgen:
+vfsgen: data/
+	go get github.com/shurcooL/vfsgen
```

Comment:
This should eventually be baked into a docker image for reproducible builds. This pattern is used elsewhere in istio/istio to auto-generate code (e.g. see https://github.com/istio/istio/pull/7343). https://github.com/istio/istio/blob/master/docker/Dockerfile.go_generate_dependency exists and solves a similar problem but encoding files into golang code. However it uses github.com/jteeuwen/go-bindata/ which is not maintained. As we integrate parts of iop into istioctl it would be good to settle on a common package for encoding data files into istio binaries.